commander/src/bootstrapping/commands/generator/export.ts

```diff
@@ -119,8 +121,14 @@ export abstract class ExportCommand extends BaseIPCClientCommand {
 
 		if (flags.output) {
 			fs.writeJSONSync(flags.output, output, { spaces: ' ', mode: OWNER_READ_WRITE });
+			this.log(`Generator info is exported to ${flags.output}`);
 		} else {
-			this.log(JSON.stringify(output, undefined, '  '));
+			fs.writeJSONSync(`${process.cwd()}/genInfo.json`, output, {
```

Comment:
Instead of invoking `fs.writeJSONSync()` twice in each of the 2 possible scenarios, the `if` part could be used to only store an export path to a variable, and then after `if` you could pass the variable as the first argument to a single call to `fs.writeJSONSync()`.

Reply:
Maybe 👇 is more optimal ...

```ts
const outputPath = flags.output || `${process.cwd()}/genInfo.json`;
fs.writeJSONSync(outputPath, output, { spaces: ' ', mode: OWNER_READ_WRITE });
this.log(`Generator info is exported to ${outputPath}`);

Reply:
Yes, that's what I had in mind, but I didn't explain it very well. Not invoke twice, but invoke from 2 different `if` sections.